arangod/IResearch/IResearchRocksDBRecoveryHelper.cpp

```diff
@@ -112,13 +112,17 @@ bool IResearchRocksDBRecoveryHelper::skip(Impl& impl) {
 
 template<bool Force>
 void IResearchRocksDBRecoveryHelper::clear() noexcept {
-  if constexpr (!Force) {
+  if constexpr (Force) {
+    if (!_ranges.empty()) {
+      _ranges.clear();
+    }
```

Comment:
A little optimization for case when few IndexCreate/IndexDrop happens in row

Reply:
Ok, but with the PR's changes we are clearing `_ranges` under different conditions as before. 
Which may be fine, but I don't know.

Reply:
If `Force` is true, we are now clearing unconditionally. Previously we only cleared `_ranges` if the other if condition was true.

Reply:
Because if force is true it could be empty, in such case clear is unnecessary, because it's already empty.
But if force is false it's never empty (after not force check) so we don't need additional check